client/src/containers/MultiFactorApp.js

```diff
@@ -9,12 +9,17 @@ import LoadingIndicator from 'components/LoadingIndicator';
 
 class MultiFactorApp extends Component {
   constructor(props) {
+    const { ss: { i18n } } = window;
+
     super(props);
     this.state = {
+      loginCompleted: false,
       schema: null,
       schemaLoaded: false,
+      title: i18n._t('MFA.TITLE', 'Multi-factor authentication'),
```

Comment:
`MultiFactorApp` maybe might be better. `MFA` is unlikely to have global collisions but you never know, it's only three letters.